docs/reference/method/MongoDBClient-createClientEncryption.txt

```diff
@@ -0,0 +1,50 @@
+=========================================
+MongoDB\\Client::createClientEncryption()
+=========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Client::createClientEncryption()
+
+   Returns a ClientEncryption object for manual encryption and decryption of
+   values.
+
+   .. code-block:: php
+
+      function createClientEncryption(array $options): MongoDB\Driver\ClientEncryption
+
+   This method has the following parameters:
+
+   .. include:: /includes/apiargs/MongoDBClient-method-createClientEncryption-param.rst
+
+   The ``$options`` parameter supports all options documented in the extension
+   manual. For the ``keyVaultClient`` option, an instance of ``MongoDB\Client``
+   is automatically unwrapped and the ``MongoDB\Driver\Manager`` instance is
```

Comment:
```
:php:`MongoDB\\Driver\\Manager <class.mongodb-driver-manager>`
```

Reply:
The suggestion above was to use `:phpclass:` instead of `php:` - is there a significant difference between these two styles?

Reply:
`:php:` is used for links to PHP.net, while `:phpclass:` is used for internal links within the PHPLIB docs. I made a typo above, so I'll fix.